src/math/math.js

```diff
@@ -258,5 +258,17 @@ pc.math = {
         x = (x - min) / (max - min);
 
         return x * x * x * (x * (x * 6 - 15) + 10);
+    },
+
+    /**
+     * @function
+     * @name pc.math.roundUp
+     * @description Rounds a number up to nearest multiple.
+     * @param {number} numToRound - The number to round up.
+     * @param {number} multiple - The multiple to round up to.
+     * @returns {number} A number rounded up to nearest multiple.
+     */
+    roundUp: function (numToRound, multiple) {
+        return Math.ceil(numToRound / multiple) * multiple;
```

Comment:
I'd still keep the handling of multiple being zero. Something like:
```
if (multiple === 0) return numToRound;
return Math.ceil(numToRound / multiple) * multiple;
```

Reply:
fantastic, thanks, done